doc/changelog/02-changed/1250-archimedean-mixin.md

```diff
@@ -0,0 +1,4 @@
+- in `archimedean.v`
+  + definition of `ceil` (use lemma `ceilNfloor` explicitly)
+    ([#1250](https://github.com/math-comp/math-comp/pull/1250),
+    by Pierre Roux).
```

Comment:
I think this contribution should be partly attributed to me.

Reply:
Arg, even entirely, this was automatically generated when adding the missing changelog and I missed it. Fixed by https://github.com/math-comp/math-comp/pull/1383 . Thanks for noticing it.